waltz-ng/client/common/colors.js

```diff
@@ -21,6 +21,11 @@ export const red = d3.rgb('#DA524B');
 export const grey = d3.rgb('#939393');
 export const blue = d3.rgb('#5271CC');
 
+export const amberBg = d3.rgb('#FCF2D7');
+export const greenBg = d3.rgb('#DFF1D2');
+export const redBg = d3.rgb('#F2D7D7');
+export const greyBg = d3.rgb('#F5F5F5');
+
```

Comment:
hmmmm .....  ;)  Okay for now I guess - suspect theres a simpler thing we can do with d3.rgb or d3.hsl ....